modules/react/common/index.ts

```diff
@@ -7,3 +7,5 @@ export * from './lib/utils';
 export * from './lib/CanvasProvider';
 export * from './lib/InputProvider';
 export * from './lib/EllipsisText';
+export * from './lib/responsive/responsiveContext';
```

Comment:
Question: Maybe don't export `isWithinBreakpoint`?

Reply:
Talked about it. Definitely don't export it.

Reply:
Updated. Only using locally in `useResponsiveContainerStyles` hook file